index.html

```diff
@@ -165,7 +185,7 @@ <h1 class="card-title">Multi-Post Sorties</h1>
         </div>
         <div class="grid-card">
             
-            <img class="card-img" src="./assets/Snapshoot4.png" alt="Card image">
+            <img class="card-img c-2" src="./assets/Snapshoot4.png" alt="Card image">
             <div class="card-body">
                 <h1 class="card-title">Multi-Post Sorties</h1>
                 <div class="card-attribut">
```

Comment:
On desktop view, you need to ensure that for each of the projects, the page width is divided.  half of it contains the project's picture and the other half contains the project's description. This has to be uniform across all projects in your portfolio. To enforce this, you might have to give equal width and length to all the images in your portfolio. Or better still you can style the container for each of the half such that they occupy exactly half of the total width. 
See the image below for better understanding.

_**Figma Design**_
![original](https://user-images.githubusercontent.com/26856491/121508460-874f2b00-c9dd-11eb-8f75-1075627a6a86.png)

_**Your Design**_
![clone](https://user-images.githubusercontent.com/26856491/121508478-8c13df00-c9dd-11eb-937f-155f88aca9a7.png)

Reply:
- [ ] Please work on this I can't see any change follow the given design

Reply:
I applied the changes directly to all the cards this is the view
![changes](https://user-images.githubusercontent.com/37901469/121674946-9cde5680-caaa-11eb-9cb3-0e74c1b54c8c.PNG)
I readjusted it